package.json

```diff
@@ -60,40 +61,40 @@
     "sat": "^0.8",
     "semantic-ui-offline": "^2.4.1-1",
     "style-loader": "^0.13.1",
-    "terser-webpack-plugin": "^1.1.0",
+    "terser-webpack-plugin": "^5.3.9",
```

Comment:
This was a big bump! Causing failures on deploy: https://app.circleci.com/pipelines/github/stride-nyc/remote_retro/347/workflows/22d311e2-2e07-43fc-a077-a898e7aa04ba/jobs/3785